spec/requests/partners_requests_spec.rb

```diff
@@ -141,6 +141,7 @@
 
     context "csv" do
       let(:response_format) { 'csv' }
+      before { create(:distribution, partner: partner) }
```

Comment:
Why was this added? It doesn't seem relevant to the PR.

Reply:
Without creating a distro for the partner, the export method that's called in the csv format fails with `nil` distributions.  Instead let me put a nil catch in the export.